Rest unit tests in `python-oslogin` fail with `Unknown field for SshPublicKey: DESCRIPTOR`

Unit tests in https://github.com/googleapis/python-oslogin/pull/249 are failing with `AttributeError: Unknown field for SshPublicKey: DESCRIPTOR`

See build log [here](https://github.com/googleapis/python-oslogin/actions/runs/5468037243/jobs/9955131991?pr=249).
```
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_create_ssh_public_key_rest_interceptors[True] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_create_ssh_public_key_rest_interceptors[False] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_get_ssh_public_key_rest_interceptors[True] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_get_ssh_public_key_rest_interceptors[False] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_update_ssh_public_key_rest_interceptors[True] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
FAILED tests/unit/gapic/oslogin_v1/test_os_login_service.py::test_update_ssh_public_key_rest_interceptors[False] - AttributeError: Unknown field for SshPublicKey: DESCRIPTOR
```

--- a/gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2
+++ b/gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2
@@ -1311,7 +1311,7 @@ def test_{{ method_name }}_rest_interceptors(null_interceptor):
         req.return_value.status_code = 200
         req.return_value.request = PreparedRequest()
         {% if not method.void %}
-        req.return_value._content = {% if method.output.ident.package == method.ident.package %}{{ method.output.ident }}.to_json({{ method.output.ident }}()){% else %}json_format.MessageToJson({{ method.output.ident }}()){% endif %}
+        req.return_value._content = {% if method.output.ident.is_proto_plus_type %}{{ method.output.ident }}.to_json({{ method.output.ident }}()){% else %}json_format.MessageToJson({{ method.output.ident }}()){% endif %}
 
         {% if method.server_streaming %}
         req.return_value._content = "[{}]".format(req.return_value._content)

--- a/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
+++ b/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
@@ -1206,7 +1206,7 @@ def test_{{ method_name }}_rest_interceptors(null_interceptor):
         req.return_value.status_code = 200
         req.return_value.request = PreparedRequest()
         {% if not method.void %}
-        req.return_value._content = {% if method.output.ident.package == method.ident.package %}{{ method.output.ident }}.to_json({{ method.output.ident }}()){% else %}json_format.MessageToJson({{ method.output.ident }}()){% endif %}
+        req.return_value._content = {% if method.output.ident.is_proto_plus_type %}{{ method.output.ident }}.to_json({{ method.output.ident }}()){% else %}json_format.MessageToJson({{ method.output.ident }}()){% endif %}
 
         {% if method.server_streaming %}
         req.return_value._content = "[{}]".format(req.return_value._content)

